### PR TITLE
contrib/cloud.google.com/go/pubsub.v1: add Option to Publish

### PR DESCRIPTION
**Pull request by hakankutluay**
Add service name and measured options to the Publish and WrapReceiveHandler

closes Datadog#1141

### CODE REVIEW

File: contrib/cloud.google.com/go/pubsub.v1/pubsub.go [hunk 1]
```diff
@@ -24,14 +24,28 @@ import (
...
-func Publish(ctx context.Context, t *pubsub.Topic, msg *pubsub.Message) *PublishResult {
-	span, ctx := tracer.StartSpanFromContext(
-		ctx,
-		"pubsub.publish",
+func Publish(ctx context.Context, t *pubsub.Topic, msg *pubsub.Message, opts ...Option) *PublishResult {
+	var cfg config
+	for _, opt := range opts {
+		opt(&cfg)
+	}
+
```
Comment (gbbr) [inline contrib/cloud.google.com/go/pubsub.v1/pubsub.go:32]: Please remove this empty line.

**Review by gbbr — APPROVED** [review]
Nice work, thanks! Just a small nit.

File: contrib/cloud.google.com/go/pubsub.v1/pubsub.go (hunk 1 above)
Comment (hakankutluay) [reply contrib/cloud.google.com/go/pubsub.v1/pubsub.go]: Thanks, just removed.

**Review by gbbr — APPROVED** [review]

**Comment by hakankutluay** [comment]
May I ask, is there anything required from me to the PR to merge?

**Comment by vigo** [comment]
when the merge will happen? :)

**Review by knusbaum — APPROVED** [review]

**Comment by hakankutluay** [comment]
Is there anything that blocks this PR to merge? @knusbaum @gbbr @Julio-Guerra

**Pull request merged**